superagi/vector_store/qdrant.py

```diff
@@ -0,0 +1,160 @@
+from __future__ import annotations
+
+import uuid
+from mimetypes import common_types
+from typing import Any, Dict, Iterable, List, Optional, Tuple, Sequence, Union
+
+from qdrant_client import QdrantClient
+from qdrant_client.http import models
+from qdrant_client.conversions import common_types
+
+from superagi.vector_store.base import VectorStore
+from superagi.vector_store.document import Document
+
+DictFilter = Dict[str, Union[str, int, bool, dict, list]]
+MetadataFilter = Union[DictFilter, common_types.Filter]
+
+def create_qdrant_client(
+) -> QdrantClient:
+    client = QdrantClient(host="localhost", port=6333)
```

Comment:
this cant be hardcoded. You need to get this from config. Should have new config QDRANT_HOST and QDRANT_PORT